toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -729,6 +730,20 @@ func InstallGrubCfg(installRoot, rootDevice, bootUUID string, encryptedRoot disk
 		return
 	}
 
+	// Configure IMA policy
+	err = setGrubCfgIMA(installGrubCfgFile, kernelCommandLine)
+	if err != nil {
+		logger.Log.Warnf("Failed to set ima_policy in grub.cfg: %v", err)
+		return
+	}
+
+	// Append any additional command line parameters
+	err = setGrubCfgAdditional(installGrubCfgFile, kernelCommandLine)
```

Comment:
nit - suggest to update function name for clarity

```suggestion
	err = setGrubCfgAdditionalCmdLine(installGrubCfgFile, kernelCommandLine)
```